crates/bevy_app/src/app.rs

```diff
@@ -70,6 +67,9 @@ pub(crate) enum AppError {
 /// }
 /// ```
 pub struct App {
+    /// List of plugins that have been added.
+    pub(crate) plugin_registry: PluginRegistry,
```

Comment:
Can we safely provide read-only access to this?

Reply:
Done, I'll probably need to add some more docs their too...